### ISSUE TITLE
`/status` API removed in Renovate CE

### ISSUE THREAD

**Issue by davidmehren** [mend/renovate-ce-ee#333]
It seems the new release 6.0.0 removed the `/status` API.

The mail I received regarding the renaming to "Renovate CE" talked about
> a set of administration APIs that provide real-time insights into the state of the server

which sound like a replacement for that API, but I could not find any documentation.

**Comment by rarkins** [comment]
Please see https://github.com/mend/renovate-ce-ee/pull/332/files

**Comment by morremeyer** [comment]
@rarkins The new `/api` endpoints are really useful to inspect the queue, however a lot of information that was previously available via `/status` is not available via any of the current `/api/` endpoints:

- `/api/jobs/queue` reports job ID, repository, reason and priority for the first 100 jobs in the queue
- `/api/tasks/queue` only returns an empty list `[]`

The old `/status` endpoint returned a much longer JSON object:

<details><summary>example /status API response</summary>

```json
{
    "status": {
        "bootDate": "2023-10-16T10:29:29.580Z",
        "jobs": {
            "lastEnqueueDate": "2023-10-16T10:32:52.522Z",
            "lastJob": {
                "installationId": 31059531,
                "params": {},
                "priority": 17,
                "reason": "master-issue-check",
                "repository": "org/repo"
            },
            "lastJobDispatchDate": "2023-10-16T10:32:52.827Z",
            "lastJobFinished": {
                "finished": "2023-10-16T10:32:42.824Z",
                "installationId": 123456789,
                "params": {},
                "priority": 17,
                "reason": "master-issue-check",
                "repository": "org/repo"
            },
            "queueLength": 0,
            "totalJobsProcessed": 2
        },
        "jobsInProgress": [
            {
                "repository": "org/repo",
                "started": "2023-10-16T10:32:52.827Z"
            }
        ],
        "scheduler": {
            "cron": "15 */2 * * *",
            "lastScheduling": null,
            "platform": "github"
        },
        "webhooks": {
            "lastWebhookReceived": "2023-10-16T10:32:52.511Z"
        },
        "worker": {
            "currentJob": {
                "installationId": 123456789,
                "params": {},
                "priority": 17,
                "reason": "master-issue-check",
                "repository": "org/repo"
            },
            "currentJobStart": "2023-10-16T10:32:52.827Z",
            "previousJob": {
                "installationId": 31059531,
                "params": {},
                "priority": 17,
                "reason": "master-issue-check",
                "repository": "org/repo"
            },
            "previousJobStart": "2023-10-16T10:31:35.375Z",
            "remediateServerEnabled": false
        }
    }
}
```

</details>

We have used the following fields from that response for our monitoring and would like to continue doing so:

- `.status.jobs.totalJobsProcessed` to alert on no jobs being processed for too long
- `.status.jobs.queueLength` to check if the queue is too long (can be done with the length of `/api/jobs/queue` now) 
- `.status.webhooks.lastWebhookReceived` to verify that webhooks are arriving

To make a long story short, a simple status API that exposes this information or even all the information `/status` did in the past would be very helpful.

For our concerns, exposing these in a Prometheus Metrics endpoint would be the icing on the cake and resolve #221 at the same time, but getting `/status` back one way or another would serve our use case perfectly fine, too.

**Comment by Gabriel-Ladzaretti** [comment]
status api is currently WIP, will be released pretty soon.

**Comment by Gabriel-Ladzaretti** [comment]
fixed by release 6.4.0
https://github.com/mend/renovate-ce-ee/blob/main/docs/api.md#status
